### PR TITLE
https://issues.redhat.com/browse/ACM-3901 S3 AWS train 1

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Review by chenz4027 — APPROVED** [review]
/lgtm Thanks Brandi!

File: clusters/release_notes/whats_new.adoc [hunk 1]
```diff
@@ -23,22 +23,7 @@ If you installed {ocp-short} or {product-title-short}, you automatically receive
...
-See xref:../api/api_intro.adoc#apis[APIs] for more information.
-
-- You can use two control plane types: hosted and standalone. _Standalone_ is the console wizard feature and _Hosted_ provides to specific steps and guidance so that you can create the cluster from your CLI.
-
-- You can now update the `MultiClusterEngine` custom resource to specify whether a hub cluster is managed by itself. See xref:../install_upgrade/adv_config_install.adoc#local-cluster[Local cluster enablement] for more information.
-
-- You can modify the settings in the `ConfigMap` and `AgentServiceConfig` files to specify unauthenticated registries when creating an infrastructure environment in a disconnected environment. See xref:../cluster_lifecycle/create_infra_env.adoc#enable-cim[Enabling the Central Infrastructure Management service] for more information.
-
-- Scaling with MachinePool is now generally available so you can easily configure autoscaling to scale your resources. See xref:../cluster_lifecycle/scale_machinepool.adoc#scaling-machinepool[Scaling with MachinePool] for more information.
+- You can now create an Amazon Simple Storage Service (S3) secret from *Credentials* in the console.
```
Comment (dockerymick) [inline clusters/release_notes/whats_new.adoc]:
```suggestion
- You can now create an Amazon Simple Storage Service (S3) secret from *Credentials* in the console. See xref:../clusters/credentials/credential_aws.adoc#aws_s3_create[Creating an S3 secret].

- You can now add hosts to an infrastructure environment with iPXE. See xref:../clusters/cluster_lifecycle/scale_hosts_infra_env.adoc#scale-hosts-infrastructure-env[Scaling hosts to an infrastructure environment] for more information.
```

File: release_notes/whats_new.adoc [hunk 2]
```diff
@@ -27,6 +27,7 @@ Cluster lifecycle components and features are within the {mce-short}, which is a
 View release notes, as well as tasks and support information at link:../clusters/cluster_mce_overview.adoc#cluster_mce_overview[Cluster lifecycle overview].
 
 * You can now add hosts to an infrastructure environment with iPXE. See link:../clusters/cluster_lifecycle/scale_hosts_infra_env.adoc#scale-hosts-infrastructure-env[Scaling hosts to an infrastructure environment] for more information.
+//this is part of MCE, I believe -- bcs 4/13
```
Comment (dockerymick) [inline release_notes/whats_new.adoc:30]: Made suggestion based on your comment

**Review by dockerymick — APPROVED** [review]
This looks good to me! I made a couple of suggestions. There a couple of inline links, not sure if you prefer to handle that in this PR or a separate one

**Pull request merged**